extension/app/scripts.babel/background.js

```diff
@@ -64,11 +64,11 @@ function createResultsHTML(results) {
     item.score.subItems.forEach(subitem => {
       const debugString = subitem.debugString ? ` title="${escapeHTML(subitem.debugString)}"` : '';
 
-      // TODO: make this work with numeric values.
       const status = subitem.value ?
           `<span class="pass" ${debugString}>Pass</span>` :
           `<span class="fail" ${debugString}>Fail</span>`;
-      groupHTML += `<li>${escapeHTML(subitem.description)}: ${status}</li>`;
+      const rawValue = subitem.rawValue ? `(${escapeHTML(subitem.rawValue)})` : '';
```

Comment:
yeah, if this is a number, for instance, that's no good. You could put the call to `escapeHTML` around the template string instead